proceeding/tables/survey_table.tex

```diff
@@ -0,0 +1,23 @@
+%table
+
+\begin{table*}[h]
+
+\caption{Survey information.}
+\label{tab:a}
+\tabcolsep7pt\begin{tabular}{ || c|cc ||}
+\hline
+
+\textbf{Survey name} & \textbf{Telescope/Observatory} & \textbf{Frequency band}\\ \hline
```

Comment:
I think this set of columns might work better:
- Image (what's Survey name now)
- Resolution (roughly indicate angular resolution in deg or arcmin, I can help with that)
- Band (or "Wavelength" or "Energy")
- Color? (is it color or grayscale)
- Description -- put the Telescope/Observatory info here.

What do you think?
Have resolution, band and color as separate columns?
Or have them part of the description sentence?